### ISSUE TITLE
More recent version of LibGD

### ISSUE THREAD

**Issue by vendeeglobe** [ApacheFriends/xampp-build#22]
Is there a technical reason to not add LibGD 2.3.3 to the package?

Especially for WebP and AVIF support in PHP 8.2.

**Comment by beltran-rubo** [comment]
Right. Note even LibDG can be updated, the PHP 8.2.0 included a GD bundled and the version is 2.1.0. Contributions to the project are welcome!
>GD Version => bundled (2.1.0 compatible)
